### PR TITLE
Add Makefiles for handling transifex translations

### PR DESCRIPTION
**Pull request by dschmidt**

### CODE REVIEW

File: admin/win/nsi/l10n/Makefile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#
+# Translation
+#--------------------------------------
+
+.PHONY: l10n-push
+l10n-push: ;
+	tx -d push -s --skip --no-interactive
+
+.PHONY: l10n-pull
+l10n-pull: ;
```
Comment (tboerger) [inline admin/win/nsi/l10n/Makefile]: ; should be dropped?

File: admin/win/nsi/l10n/Makefile [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#
+# Translation
+#--------------------------------------
+
+.PHONY: l10n-push
+l10n-push: ;
```
Comment (tboerger) [inline admin/win/nsi/l10n/Makefile]: Same here?

File: admin/win/nsi/l10n/Makefile (hunk 1 above)
Comment (dschmidt) [reply admin/win/nsi/l10n/Makefile]: No, this is how you specify an empty recipe: https://www.gnu.org/software/make/manual/html_node/Empty-Recipes.html

File: admin/win/nsi/l10n/Makefile (hunk 2 above)
Comment (dschmidt) [reply admin/win/nsi/l10n/Makefile]: Same answer as above ;-)

File: admin/win/nsi/l10n/Makefile (hunk 1 above)
Comment (dschmidt) [reply admin/win/nsi/l10n/Makefile]: Ah, yes. That was for debugging. Good catch!

**Comment by dschmidt** [comment]
Wtf?! I did not close this and I cannot reopen it now o.O

**Pull request merged**